documentation/en/default-lotus-config.toml

```diff
@@ -260,22 +253,14 @@
   # env var: LOTUS_EVENTS_DISABLEREALTIMEFILTERAPI
   #DisableRealTimeFilterAPI = false
 
-  # DisableHistoricFilterAPI will disable the HistoricFilterAPI that can create and query filters for actor events
-  # that occurred in the past. HistoricFilterAPI maintains a queryable index of events.
-  # The API is enabled when Fevm.EnableEthRPC or EnableActorEventsAPI is true, but can be disabled selectively with this flag.
-  #
-  # type: bool
-  # env var: LOTUS_EVENTS_DISABLEHISTORICFILTERAPI
-  #DisableHistoricFilterAPI = false
-
   # EnableActorEventsAPI enables the Actor events API that enables clients to consume events
   # emitted by (smart contracts + built-in Actors).
   # This will also enable the RealTimeFilterAPI and HistoricFilterAPI by default, but they can be
   # disabled by setting their respective Disable* options.
   #
   # type: bool
   # env var: LOTUS_EVENTS_ENABLEACTOREVENTSAPI
-  #EnableActorEventsAPI = false
+  #EnableActorEventsAPI = true
```

Comment:
Similar, do we need to change the default right now?

Reply:
No we do not. Reverted.